app/src/main/res/layout/screen_scan_simple.xml

```diff
@@ -27,4 +30,27 @@
     app:layout_constraintTop_toTopOf="parent"
     app:srcCompat="@drawable/ic_scan_qr_viewfinder" />
 
+  <androidx.cardview.widget.CardView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="8dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginStart="8dp"
+    android:background="@color/white100"
+    android:visibility="gone"
+    app:layout_constraintBottom_toBottomOf="@+id/qrCodeScannerView"
```

Comment:
This should just be `@id/qrCodeScannerView`, right?

Reply:
It is safer to add `@+id`.

Reply:
`@id/qrCodeScannerView` will give a compilation error if the element is missing from the layout whereas `@+id` wouldn't (I'm guessing). However, at least nowadays we use `@+id/` everywhere. @vinaysshenoy do you have any reasons why we should use it?

Reply:
Lint was flagging this as errors in my IDE, and they went away when I removed the extra `+` symbol.

Let's leave the `+` symbol in now and figure out what to do about this later.